client/src/components/Profile/EmploymentHistory/EmploymentHistoryFormItem.tsx

```diff
@@ -0,0 +1,103 @@
+import { Checkbox, Form, Input, DatePicker, Space, Button, FormInstance, Divider } from 'antd';
+import { DeleteOutlined } from '@ant-design/icons';
+import { useState } from 'react';
+import EmploymentHistoryDisplayItem from './EmploymentHistoryDisplayItem';
+import { EmploymentRecordFormItem } from '../EmploymentCard';
+import { CheckboxChangeEvent } from 'antd/es/checkbox';
+import { LocationSelect } from 'components/Forms';
+import { Location } from 'api';
+
+type EmploymentHistoryFormItemPros = {
+  name: number;
+  restField: { fieldKey?: number | undefined };
+  remove: (index: number | number[]) => void;
+  form: FormInstance<{ employmentHistory: EmploymentRecordFormItem[] }>;
+};
+
+const EmploymentHistoryFormItem = ({ name, restField, remove, form }: EmploymentHistoryFormItemPros) => {
+  const { title, dateFrom, dateTo, toPresent, companyName, officeLocation } =
+    form.getFieldValue(['employmentHistory', name]) ?? {};
```

Comment:
why do you need explicitly get this data rather than us form items to consume it automatically?